.github/workflows/cd-storybook.yml

```
@@ -2,11 +2,10 @@ name: Deploy Storybook
 
 on:
   # Runs on pushes targeting the default branch
-  # !! Uncomment the lines below to enable this workflow
-  # push:
-  #   branches: ["main"]
-  #   paths:
-  #     - frontend/**
+  push:
+    branches: ["main"]
+    paths:
+      - frontend/**
```

```suggestion
      - frontend/.storybook/**
      - frontend/src/**
      - frontend/stories/**
      - .github/workflows/cd-storybook.yml
```
Maybe we don't need to rebuild storybook for all frontend repo changes just storybook relevant ones?

---

I'm okay leaving it as `frontend/**`. Almost every folder has some relation to Storybook. `/public` houses images and locale strings, so we'd want that too. We likely want to integrate the Storybook test runner add-on. There's not much else at that point that warrants getting this granular. Going to skip committing this, but open to adding it.